### PR TITLE
Add uptime dashboard to docs

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

File: docs/reference/networks.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Specs for Stellar's various networks
 
 :::info
 
-To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/).
+To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/) and the [Stellar Status](https://status.stellar.org/) for network and service uptime.
```
Comment (ElliotFriend) [inline docs/reference/networks.mdx:9]:
```suggestion
To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/) and the [Stellar Statuspage](https://status.stellar.org/) for network and service uptime.
```

Comment (janewang) [reply docs/reference/networks.mdx:9]: I think statuspage is two words.

Comment (ElliotFriend) [reply docs/reference/networks.mdx]: I thought so, too. But the atlassian product that is running the status website calls itself "statuspage" 🤷‍♂️

**Review by ElliotFriend — APPROVED** [review]
I like it better like this, anyway 🤣

**Pull request merged**